src/main/scala/io/opentargets/etl/backend/target/Safety.scala

```diff
@@ -101,28 +102,27 @@ object Safety extends LazyLogging {
 
   private def transformTargetSafety(df: DataFrame): DataFrame = {
     logger.debug("Transforming target safety safety risk data.")
-    df.filter(hecatosFilter)
-      .select(
-        col("ensemblId") as "id",
-        when(col("ref").contains("Force"), "heart disease")
-          .when(col("ref").contains("Lamore"), "cardiac arrhythmia") as "event",
-        when(col("ref").contains("Force"), "EFO_0003777")
-          .when(col("ref").contains("Lamore"), "EFO_0004269") as "eventId",
-        struct(
-          col("biologicalSystem") as "tissueLabel",
-          col("uberonId") as "tissueId",
-          lit(null) as "cellLabel",
-          lit(null) as "cellFormat",
-          lit(null) as "cellId"
-        ) as "biosample",
-        col("ref") as "datasource",
-        col("pmid") as "literature",
-        struct(
-          lit(null) as "name",
-          col("liability") as "description",
-          lit(null) as "type"
-        ) as "study"
-      )
+    df.select(
+      col("ensemblId") as "id",
+      when(col("ref").contains("Force"), "heart disease")
+        .when(col("ref").contains("Lamore"), "cardiac arrhythmia") as "event",
+      when(col("ref").contains("Force"), "EFO_0003777")
+        .when(col("ref").contains("Lamore"), "EFO_0004269") as "eventId",
+      struct(
+        col("biologicalSystem") as "tissueLabel",
+        col("uberonId") as "tissueId",
+        lit(null) as "cellLabel",
```

Comment:
@JarrodBaker, this is a bit tricky as lit infers the type from the parameter and null is Null which is at the bottom of the type hierarchy as a ref type as Nothing as a type. I would try either cast to the intended type of use the scala spark lang functions for it like `lit(null).cast(StringType)` or `typedLit[String](null)`

```Scala
@ typedLit[String](null) 
res0: Column = CAST(NULL AS STRING)

@ lit(null).cast(StringType) 
res1: Column = CAST(NULL AS STRING)
```